x-pack/platform/plugins/shared/onechat/public/application/context/conversation/embeddable_conversations_provider.tsx

```diff
@@ -38,12 +40,26 @@ export const EmbeddableConversationsProvider: React.FC<EmbeddableConversationsPr
     [coreStart, services.startDependencies]
   );
 
+  const { resolvedConversationId, isLoading: isResolvingConversation } = useResolveConversationId({
+    newConversation: contextProps.newConversation,
+    conversationId: contextProps.conversationId,
+    sessionTag: contextProps.sessionTag,
+    agentId: contextProps.agentId,
+  });
```

Comment:
We can't call this hook here without passing the necessary context that it needs.
`useResolveConversationId` --> `useLastConversationId` --> `useConversationList` --> `useOnechatServices` (this doesn't exist yet because it's the **_children_** of this provider that's wrapped with the OneChat service provider - this is why on L88 we pass `services.conversationsService` to the `useConversationsActions` hook

<img width="821" height="98" alt="Image" src="https://github.com/user-attachments/assets/b3b9622e-6c82-4a49-a3f8-e2470d6ea5b4" />